.spectral.yml

```diff
@@ -0,0 +1,139 @@
+extends: [spectral:oas2]

```

Comment:
Just in case you wanted to target your own rules only against OAS2, you can use `formats: [oas2]`.